### PR TITLE
Small improvements to wording and internal linking

### PR DESCRIPTION
**Pull request by johnwaterworth**
Improve some wording. Add internal links to new section.

### CODE REVIEW

File: src/user-research/choosing-and-using-analysis-and-synthesis-methods.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ The purpose of analysis and synthesis (sometimes also called sensemaking) is to
 
 We do at least an initial analysis and synthesis at the end of each research activity, so that we can quickly [learn, share and adapt](https://playbook.dxw.com/user-research/#user-research-principles).
 
-Later we may combine different kinds of data and findings from different research activities to create a coherent picture.
+Later we may [combine different kinds of data and findings](#rich-data-and-big-data) from different research activities to create a coherent picture.
 
 And we plan the pace of our research to avoid building up analysis and synthesis debt.
 
-Our rule of thumb is “one part research to two parts communication”. For a day spent doing interviews, for example, we expect to spend the next two days on analysis and synthesis, and sharing your findings.
+Our rule of thumb is “one part research to two parts communication”. For a day spent doing interviews, for example, we expect to spend the two more days on analysis and synthesis, and sharing findings.
```
[the remark this answers is not in the document]
Comment (johnwaterworth) [reply src/user-research/choosing-and-using-analysis-and-synthesis-methods.md]: I'd already removed the stray 'the'. And I've removed the unnecessary 'more'.

**Review by ZuleimaMR — COMMENTED** [review]
in line 26 I still see the word 'more'
"Our rule of thumb is “one part research to two parts communication”. For a day spent doing interviews, for example, we expect to spend two more days on analysis and synthesis, and sharing findings."

In line 30, why are you keeping the word 'findings' at the start of the sentence? "findings and next steps we can justify - as we are often challenged to explain why we have come to those conclusions". Findings are not conclusions and this point seems to be about 'next steps' not findings. Next steps are an outcome of analysis of findings.

**Comment by johnwaterworth** [comment]
Got rid of the 'more' again. That change must have got lost in the Github changes yesterday.

**Comment by johnwaterworth** [comment]
This point is about being able to justify both findings and next steps. As we get challenged on both.
A dictionary definition of conclusions is "a judgement or decision reached by reasoning". So I'm OK using conclusion in this context.

**Review by ZuleimaMR — APPROVED** [review]

**Pull request merged**